libraries/protocol/smt_operations.cpp

```diff
@@ -35,18 +22,20 @@ void smt_create_operation::validate()const
       ("prec1",symbol.decimals())("prec2",precision) );
 }
 
-bool is_valid_unit_target( const account_name_type& name )
+bool is_valid_unit_target( const unit_target_type& name )
```

Comment:
This needs to return true for `$alice.vesting`, and currently does not.

Reply:
This is not implemented yet.